kotti/alembic/versions/559ce6eb0949_update_node_path_column.py

```diff
@@ -13,13 +13,10 @@
 
 def upgrade():
 
-    from kotti import DBSession
-    from kotti.resources import Node
-
-    for node in DBSession.query(Node).with_polymorphic([Node]):
-        # append '/' to all nodes but root
-        if node.path != u'/':
-            node.path += u'/'
+    from kotti.resources import DBSession
+    DBSession.execute(
+        "update nodes set path = path || '/' where path not like '/'"
```

Comment:
I think this should be ``update nodes set path = path || '/' where path not like '%/'`` (append a slash to every path that doesn't already end with ``/``).